src/main/java/com/uid2/shared/auth/ClientKey.java

```diff
@@ -25,8 +27,9 @@ public class ClientKey implements IRoleAuthorizable<Role> {
     private int siteId;
     private boolean disabled;
 
-    public ClientKey(String key, String secret, String name, String contact, Instant created, Set<Role> roles, int siteId, boolean disabled) {
+    public ClientKey(String key, String keyHash, String secret, String name, String contact, Instant created, Set<Role> roles, int siteId, boolean disabled) {
```

Comment:
I'm still sort of keen to have the salt as its own property.. 

Is there a strong need to do extra processing to parse out the salt from the same field?